<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="wrapper">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="information">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Все книги</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QListWidget" name="listWidget_1"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Добавить книгу</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QComboBox" name="addSelectType">
               <item>
                <property name="text">
                 <string>Книга</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Журнал</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="addlabel">
               <property name="text">
                <string>Введите поля для новой книги:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addName">
               <property name="placeholderText">
                <string>Название книги</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addAuthorN">
               <property name="placeholderText">
                <string>Имя автора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="addAuthorS">
               <property name="placeholderText">
                <string>Фамилия автора</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="addDate"/>
             </item>
             <item>
              <widget class="QLineEdit" name="addTheme">
               <property name="placeholderText">
                <string>Тема журнала</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="addNumber"/>
             </item>
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Добавить книгу</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <attribute name="title">
           <string>Редактировать книгу</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="6" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Отредактировать</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QDateEdit" name="dataEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="authorEdit1">
               <property name="placeholderText">
                <string>Имя автора</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="nameEdit">
               <property name="placeholderText">
                <string>Название книги</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLineEdit" name="authorEdit2">
               <property name="placeholderText">
                <string>Фамилия автора</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLineEdit" name="themeEdit">
               <property name="placeholderText">
                <string>Тема журнала</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QSpinBox" name="numberEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <attribute name="title">
           <string>Удалить книгу</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="deleteInfo">
               <property name="text">
                <string>Вы действительно хотите удалить данную книгу?</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteName">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteAuthorN">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteAuthorS">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteDate">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteTheme">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="deleteNumber">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton">
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="menu">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Сохранить в файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Загрузить из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSort">
         <property name="text">
          <string>Отсортировать по имени</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_3">
         <property name="maximumSize">
          <size>
           <width>292</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonExit">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
